app/_hub/kong-inc/zipkin/index.md

```diff
@@ -60,51 +63,62 @@ params:
+        The length in bytes of each request's Trace ID. The value can be either `8` or `16`.
     - name: header_type
       required: true
       default: preserve
+      datatype: string
       description: |
         All HTTP requests going through the plugin will be tagged with a tracing HTTP request.
         This property codifies what kind of tracing header the plugin expects on incoming requests.
-        Possible values are `b3`, `b3-single`, `w3c`, or `preserve`. The `b3` option means that
+        Possible values are `b3`, `b3-single`, `w3c`, `preserve`, `jaeger`, or `ot`. The `b3` option means that
         the plugin expects [Zipkin's B3 multiple headers](https://github.com/openzipkin/b3-propagation#multiple-headers)
         on incoming requests, and will add them to the transmitted requests if they are missing from it.
         The `b3-single` option expects or adds Zipkin's B3 single-header tracing headers.
         The `w3c` option expects or adds W3C's traceparent tracing header. The `preserve` option
-        does not expect any format, and will  transmit whatever header is recognized or present,
-        defaulting to `b3` if none is found. In case of mismatch between the expected and incoming
+        does not expect any format, and will transmit whatever header is recognized or present,
+        with a default of `b3` if none is found. In case of a mismatch between the expected and incoming
         tracing headers (for example, when `header_type` is set to `b3` but a w3c-style tracing header is
         found in the incoming request), then the plugin will add both kinds of tracing headers
-        to the request and generate a mismatch warning in the logs.
+        to the request and generate a mismatch warning in the logs. `jaeger` will use and expect
+        [Jaeger-style tracing headers](https://www.jaegertracing.io/docs/1.22/client-libraries/#propagation-format) (`uber-trace-id`).
+        The `ot` option is for [OpenTelemetry tracing headers](https://github.com/open-telemetry/opentelemetry-java/blob/96e8523544f04c305da5382854eee06218599075/extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/OtTracerPropagator.java) of the form `ot-tracer-*`.
     - name: default_header_type
       required: true
       default: b3
+      datatype: string
       description: |
         Allows specifying the type of header to be added to requests with no pre-existing tracing headers
         and when `config.header_type` is set to `"preserve"`.
-        When `header_type` is set to any other value, `default_header_type` is ignored.
+        When `header_type` is set to any other value, `default_header_type` is ignored. Possible values are
+        `b3`, `b3-single`, `w3c`, `jaeger`, or `ot`.
+    - name: tags_header
+      required: true
+      default: Zipkin-Tags
+      datatype: string
+      description: |
+        XX REVIEWERS need a description
```

Comment:
The Zipkin plugin will add extra headers to the tags associated with any HTTP requests that come with a header named as configured by this property. The format is `name_of_tag=value_of_tag`, separated by commas. For example: with the default value, a request with the header `Zipkin-Tags: fg=blue, bg=red` will generate a trace with the tag `fg` with value `blue`, and another tag called `bg` with value `red`.